sys/dev/hwpmc/hwpmc_arm64_md.c

```diff
@@ -120,7 +120,7 @@ pmc_save_user_callchain(uintptr_t *cc, int maxsamples,
 
 	td = curthread;
 	pc = PMC_TRAPFRAME_TO_PC(tf);
-	*cc++ = pc;
+	*cc++ = (uintptr_t)pc;
```

Comment:
This probably should be saving full capabilities in hybrid or just ptraddr_t in purecap, but both are more invasive hwpmc-wide changes and this is sufficient to get it to build (I haven't dared run it, hwpmc is scary enough without throwing CHERI into the mix).